spinalcordtoolbox/scripts/sct_dmri_moco.py

```diff
@@ -161,8 +193,28 @@ def main(argv=None):
     if arguments.param is not None:
         param.update(arguments.param)
 
+    path_qc = arguments.qc
+    qc_fps = arguments.qc_fps
+    qc_dataset = arguments.qc_dataset
+    qc_subject = arguments.qc_subject
+    qc_seg = arguments.qc_seg
+
+    if path_qc is not None:
+        if qc_seg is None:
+            raise Exception("-qc-seg is needed to ensure proper cropping around spinal cord in QC report.")
+
     # run moco
-    moco_wrapper(param)
+    fname_output_image = moco_wrapper(param)
+
+    set_global_loglevel(verbose)  # moco_wrapper seems to change verbose to 0
```

Comment:
I'm really curious about why this is. This seems worth investigating further.

Reply:
I could take a look and try to find where the verbose is changed in `moco_wrapper.` Is there a command to check the global verbose value?

Reply:
Thank you for offering! I was thinking this over last night, and think I have a rough idea of why it might be. So, I'd be happy to look into this. :slightly_smiling_face: 

I'll try and finish that task + reviewing `slice.py` by the end of today.

Reply:
Aha. `moco_wrapper` calls the script `sct_apply_transfo`, and that call explicitly sets `verbose=0`:

https://github.com/neuropoly/spinalcordtoolbox/blob/916ab591ca2e35cab9f5f27ec2439ce8fdb0b9d2/spinalcordtoolbox/moco.py#L768-L773

Reply:
My guess is that `sct_apply_transfo` has `-v 0` because it would be too noisy otherwise. (`sct_apply_transfo` gets called repeatedly here)

The problem is, calling `sct_apply_transfo` has the side-effect of changing the global logging level... I'll open a separate issue for this.

Reply:
Done in https://github.com/neuropoly/spinalcordtoolbox/issues/3341.

@claragiroux Perhaps it could be good to update the `# moco_wrapper seems to change verbose to 0` comments to reference this new issue?

Reply:
Thanks for the investigation! I'll add the comment to reference this issue when calling `set_global_loglevel(verbose)`.